docs/advanced-configurations.md

```diff
@@ -112,6 +113,25 @@ key:
 
 To attach NowSecure SBOM data into Dependency Insights, in the `with:` section of the `convert-sarif` action, add `enable_dependencies: true`.
 
+## Custom Build Version Strings
+
+A custom version string can be attached to a build uploaded for analysis, overriding the version string contained in the package file. 
+The custom string will be displayed in the "Build" column of the application list in Platform.
```

Comment:
Looking at the screenshot QA provided - the custom tag is displayed in the `version` column in Platform UI